app/services/decidim/vocdoni/vocdoni_client.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Vocdoni
+    class VocdoniClient
+      attr_reader :wallet, :api_endpoint_env
+
+      def initialize(wallet:, api_endpoint_env:)
+        @wallet = wallet
+        @api_endpoint_env = api_endpoint_env
+      end
+
+      def fetch_election(vocdoni_election_id)
+        fetch_from_api("/elections/#{vocdoni_election_id}")
+      end
```

Comment:
`:wallet` isn't used, and `:api_endpoint_env`, as I mentioned in other comments, can be inferred from `Decidim::Vocdoni` already.  Also I think it's better to just send the `:vocdoni_election_id` in the initializer. This is what I have in mind:

```suggestion
      attr_reader :vocdoni_election_id

      def initialize(vocdoni_election_id:)
        @vocdoni_election_id = vocdoni_election_id
      end

      def fetch_election
        fetch_from_api("/elections/#{vocdoni_election_id}")
      end
```